platform/rct/hal/LED_stm32/led_device_stm32.c

```diff
@@ -43,9 +43,9 @@ static struct led_data {
         uint16_t mask;
         bool level;
 } leds[] = {
-        {LED_ERROR,     GPIO_Pin_2, 0},
+        {LED_ERROR,     GPIO_Pin_0, 0},
         {LED_GPS,       GPIO_Pin_1, 0},
-        {LED_TELEMETRY, GPIO_Pin_0, 0},
+        {LED_TELEMETRY, GPIO_Pin_2, 0},
```

Comment:
So the new board has different mappings for the LEDs?  Effectively the pins swapped around?